.github/workflows/ci.yml

```diff
@@ -10,11 +10,24 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         java: ['8', '11', '15']
     steps:
     - uses: actions/checkout@v2
-    - name: ${{ matrix.java }}
-      uses: actions/setup-java@v1
+    - uses: actions/setup-java@v2
       with:
+        distribution: 'zulu'
         java-version: ${{ matrix.java }}
+    - name: compile and install
+      run: ./mvnw install -B -V -DskipTests -Dair.check.skip-all
+    - name: test
+      run: ./mvnw install -B
+    - name: tempto test (exclusion)
```

Comment:
I think this can be converted into a matrix but it's not straightforward so I'll try that in a later PR.